docs/configuration.md

```diff
@@ -0,0 +1,109 @@
+# Configuration
+
+The packages of the JSON RPC Relay support loading of configuration from an `.env` file or via the
+environment.
+
+## Default Values
+
+The default configuration allows users to quickly get up and running without having to configure anything. This provides
+ease of use at the trade-off of some insecure default configuration. Most configuration settings have appropriate
+defaults and can be left unchanged. It is recommended to browse the properties below and adjust to your needs.
+
+A few properties omit a default value as they relate to account information and Hedera environment details.
+These properties are noted below and should be custom set per deployment.
+
+- `CHAIN_ID`
+- `HEDERA_NETWORK`
+- `MIRROR_NODE_URL`
+- `OPERATOR_ID_MAIN`
+- `OPERATOR_KEY_MAIN`
+- `SERVER_PORT`
+
+## Server
+
+The following table lists the available properties along with their default values for the [Server package](/packages/server/). 
+Unless you need to set a non-default value, it is recommended to only populate overridden properties in the custom `.env`.
+
+| Name                          | Default       | Description                                                   |
+|-------------------------------|---------------|---------------------------------------------------------------|
+| `CHAIN_ID`                    | ""            | The network chain id. Local and previewnet envs should use `0x12a` (298). Previewnet, Testnet and Mainnet should use `0x129` (297), `0x128` (296) and `0x127` (295) respectively. |
+| `HBAR_RATE_LIMIT_DURATION`    | "60000"       | hbar budget limit duration. This creates a timestamp, which resets all limits, when it's reached. Default is to 60000 (1 minute).   |
+| `HBAR_RATE_LIMIT_TINYBAR`    | "5000_000_000" | total hbar budget in tinybars.   |
+| `HEDERA_NETWORK`              | ""                         | Which network to connect to. Automatically populates the main node & mirror node endpoints. Can be `MAINNET`, `PREVIEWNET`, `TESTNET` or `OTHER`   |
```

Comment:
We give this example below.  `HEDERA_NETWORK={"127.0.0.1:50211":"0.0.3"}`.
Can we highlight what this means when the `HEDERA_NETWORK` is `OTHER`

Reply:
Done